krun.py

```diff
@@ -265,7 +265,12 @@ def inner_main(mailer, on_first_invocation, config, args):
             util.fatal(error_msg)
 
         debug("Using pre-recorded initial temperature readings")
-        platform.starting_temperatures = current.starting_temperatures
+        manifest = ManifestManager(config, platform)
+
+        platform_temps = {}
```

Comment:
I don't know. I don't see why we shouldn't use the concise syntax. You wouldn't say `dict((1, 2), (3, 4))`, you'd use `{1: 2, 3:4}`, so why `dict()`?